### ISSUE TITLE
[Effect Suggestion] I'm Ready for My Close Up

### ISSUE THREAD

**Issue by CRB9000** [gta-chaos-mod/ChaosModV#3348]
**I'm Ready For My Close Up**
Potential name(s) for the effect
I'm Ready For My Close Up
All Eyes On Me
It's All About Me

**Duration (choose one and replace the corresponding "[ ]" with "[X]")**
- [X] Long
- [ ] Short
- [ ] No duration

**Effect Description**
Camera locks on player's face close from front. No first person.

**Comment by anto4440** [comment]
isnt that just binoculars?

**Comment by CRB9000** [comment]
No. Binocular is the player's POV. This turns the camera on the player's face, doesn't have to be zoomed. Like a 3rd person view but instead of an over the shoulder it's more like you are wearing a selfie cam.

**Comment by MoneyWasted** [comment]
Closing as similar to the existing `PlayerLockCamera` / `PlayerZoomZoomCam` effects.
